types/v56/types.go

```diff
@@ -475,10 +476,10 @@ type Task struct {
 type CapacityWithUsage struct {
 	Units     string `xml:"Units"`
 	Allocated int64  `xml:"Allocated,omitempty"`
-	Limit     int64  `xml:"Limit,omitempty"`
+	Limit     int64  `xml:"Limit"`
 	Reserved  int64  `xml:"Reserved,omitempty"`
 	Used      int64  `xml:"Used,omitempty"`
-	Overhead  int64  `xml:"Overhead,omitempty"`
+	Overhead  int64  `xml:"Overhead,omitempty"` // not available anymore from API v30.0
```

Comment:
More of a question - we're using v27.0, how did you spot v30.0? Was it only in docs or did you hit it somehow?

Reply:
Accidentally spotted checking against API:) from time to time I shuffle between versions